src/pages/defi/uniswap-v2-add-remove-liquidity/UniswapV2Liquidity.sol

```diff
@@ -13,10 +14,20 @@ contract TestUniswapLiquidity {
         uint _amountB
     ) external {
         IERC20(_tokenA).transferFrom(msg.sender, address(this), _amountA);
-        IERC20(_tokenB).transferFrom(msg.sender, address(this), _amountB);
+        // IERC201(_tokenB).transferFrom(msg.sender, address(this), _amountB);
+        // Manualy transfer from USDT
+        (bool successTransferFrom, ) = address(_tokenB).call(
+            abi.encodeCall(IERC20.transferFrom, (msg.sender, address(this), _amountB))
+        );
+        require(successTransferFrom, "USDT transferFrom failed");
 
         IERC20(_tokenA).approve(ROUTER, _amountA);
-        IERC20(_tokenB).approve(ROUTER, _amountB);
```

Comment:
Similar to comment above about `safeTransferFrom`

Reply:
Thanks, added